### PR TITLE
Document how to run local dev

### PR DESCRIPTION
**Pull request by shawndwells**
documents how to run local dev environment assuming nothing has been installed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,36 @@ This is the code behind https://crowdstrike.github.io/community/
 On that webpage, you can find content such as:
 - [List of CrowdStrike's Open Source Projects](https://crowdstrike.github.io/community/)
 - [Guidance for CrowdStrike Engineering](https://crowdstrike.github.io/community/open-source-policy/#engineering)
-- [Guidance for All Other Departments/Orgs](https://crowdstrike.github.io/community/open-source-policy/#everyone-else)
\ No newline at end of file
+- [Guidance for All Other Departments/Orgs](https://crowdstrike.github.io/community/open-source-policy/#everyone-else)
+
+
+
+## Developing the Site
```
Comment (redhatrises) [inline README.md:11]:
```suggestion
## Building

To build from the repo after cloning:
```

File: README.md [hunk 2]
```diff
@@ -4,4 +4,36 @@ This is the code behind https://crowdstrike.github.io/community/
...
+
+1. Ensure ``bundler`` and `jekyll` and ``rake`` are installed locally:
+```shell
+gem install bundler jekyll rake
+```
+2. Install NPM dependencies. The webroot is the `docs` folder, so you must change into that directory:
+```shell
+cd docs/ && npm install
+```
```
Comment (redhatrises) [inline README.md:20]:
```suggestion
2. Change directory to the `docs` folder and use `bundler` to build the site:
```shell
cd docs/ && bundle exec jekyll build
```
```

File: README.md [hunk 3]
```diff
@@ -4,4 +4,36 @@ This is the code behind https://crowdstrike.github.io/community/
...
+
+3. Start the development server locally. It will watch for chances you make, and reload (nearly) instantly:
```
Comment (redhatrises) [inline README.md:22]:
```suggestion
3. Start the development server locally. Add the `--incremental` flag if you want to watch and reload changes:
```

**Review by redhatrises — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (shawndwells) [reply README.md:20]: If you don't `npm install` then the site will fail to build (many missing modules)

File: README.md (hunk 3 above)
Comment (shawndwells) [reply README.md:22]: `bundle exec jekyll serve` automatically adds the ``--incremental`` flag

File: README.md (hunk 2 above)
Comment (redhatrises) [reply README.md:20]: Odd.... I haven't seen that behavior at all. I don't even have npm installed. Is this a macos requirement?

File: README.md (hunk 3 above)
Comment (redhatrises) [reply README.md:22]: From what I have seen, it doesn't get turned on automatically. Line 32 shows it as disabled.

Should we have instructions for linux vs macos?

File: README.md (hunk 2 above)
Comment (shawndwells) [reply README.md:20]: Dunno. I do develop on OSX, and if the node modules are missing or outdated, a bunch of errors are thrown and the site doesn't build locally.

Comment (redhatrises) [reply README.md:20]: funny... okay. I'll merge and we can fix later.

Comment (shawndwells) [reply README.md:20]: ( I can't duplicate anymore, since, well, they're all installed now)

File: README.md (hunk 1 above)
Comment (shawndwells) [reply README.md:11]: `build` eludes to a one time thing, like a compilation. Prefer "Developing" since it's more of a continuous process.

**Pull request merged**